spec/unit/knife/core/windows_bootstrap_context_spec.rb

```diff
@@ -51,10 +47,7 @@
     let(:crt_files) { ::Dir.glob(::File.join(mock_cert_dir, "*.crt")) }
     let(:pem_files) { ::Dir.glob(::File.join(mock_cert_dir, "*.pem")) }
     let(:other_files) { ::Dir.glob(::File.join(mock_cert_dir, "*")) - crt_files - pem_files }
-
-    before do
-      bootstrap_context.instance_variable_set(:@chef_config, Mash.new(trusted_certs_dir: mock_cert_dir))
-    end
+    let(:chef_config) { { trusted_certs_dir: mock_cert_dir } }
```

Comment:
A lot of these tests were setting this instance variable directly instead of passing a hash while instantiating, which it turns out was completely unnecessary.